crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -0,0 +1,964 @@
+        Ok(match item.request.payment_method_data {
+            api::PaymentMethodData::Card(ref ccard) => Ok(Self {
+                cards_payment_request: Some(PaymentRequestCards {
+                    amount: format!("{:.2}", amount),
+                    currency: item.request.currency.to_string(),
+                    pan: ccard.card_number.peek().clone(),
+                    cvv: ccard.card_cvc.peek().clone(),
+                    expiry_date: format!(
+                        "{}/{}",
+                        ccard.card_exp_month.peek().clone(),
+                        ccard.card_exp_year.peek().clone()
+                    ),
+                    cardholder: ccard.card_holder_name.peek().clone(),
+                    reference: item.payment_id.clone(),
+                    redirect_url: item.get_return_url()?,
+                    billing_city: params.billing_city,
+                    billing_country: params.billing_country,
+                    billing_street1: params.billing_street1,
+                    billing_postcode: params.billing_postcode,
+                    customer_email: item.request.email.clone(),
+                    customer_ip_address: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .ip_address,
+                    browser_accept_header: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .accept_header
+                        .clone(),
+                    browser_language: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .language
+                        .clone(),
+                    browser_screen_height: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .screen_height
+                        .clone()
+                        .to_string(),
+                    browser_screen_width: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .screen_width
+                        .clone()
+                        .to_string(),
+                    browser_timezone: item
+                        .request
+                        .browser_info
+                        .as_ref()
+                        .unwrap_or(&default_browser_info)
+                        .time_zone
+                        .clone()
+                        .to_string(),
```

Comment:
use the values from browser_info directly, if the required field error for `browser_info` is handled above, here we can directly access the fields like `browser_info. time_zone ` etc

Reply:
@saiharsha-juspay are you planning to use browser_info here?